### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by harshal0212** [community/community#142443] [GitHub Education]
### Hi everyone,

my request was submitted on Oct 21
when will I be able to use the Student developer pack?
![image](https://github.com/user-attachments/assets/6fffa9fb-b4d5-413f-80a3-62ab0be99a11)
thank You!

**Comment by webdevmahdi** [comment]
Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution. Have a verifiable school-issued email address or upload documents that prove your current student status. This is the requirement for getting a student developer pack. So, make sure you meet this requirement.

**Comment by gaur-avvv** [comment]
after 5-6 day you will be able to use GitHub student pack benefits...be patience!

**Comment by davevad93** [comment]
Hi @harshal0212 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @harshal0212 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
